www/apps/docs/content/upgrade-guides/medusa-ui/2-0-0.md

```diff
@@ -0,0 +1,71 @@
+---
+description: "Actions Required for v.2.0.0"
```

Comment:
```suggestion
description: "Actions Required for v.2.0.0"
sidebar_custom_props:
  iconName: 'component-solid'
```

Feel free to pick another icon, but make sure it's exported in `www/apps/docs/src/theme/Icon/index.tsx`